packages/rest/test/unit/backward-compatibility.test.ts

```diff
@@ -0,0 +1,18 @@
+import {get} from '../..';
+import {expect} from '@loopback/testlab';
+
+describe('backward-compatibility', () => {
+  it('exports functions from @loopback/openapi-v2', async () => {
+    try {
+      class Test {
```

Comment:
You need to add `/* tslint:disable-next-line:no-unused-variable */` to fix the style issue: